operator/test/cucumber_test.go

```diff
@@ -0,0 +1,129 @@
+package test_test
```

Comment:
this package name looks interesting :D 

anyhow, would it make sense to introduce a separate module for these tests? I think since they are e2e, they should be kept apart from our unit tests

Reply:
Agree, makes sense to move them into a separate module. Will do that...